lib/Prometheus/Tiny.pm

```diff
@@ -16,21 +16,28 @@ my $DEFAULT_BUCKETS = [
 ];
 
 sub new {
-  my ($class) = @_;
+  my ($class, %arg) = @_;
   return bless {
     metrics => {},
     meta => {},
+    default_labels => ($arg{default_labels} || {}),
```

Comment:
What do you think of doing a copy here? I'm always sorta :raised_eyebrow: about using a passed-in hashref because the caller can change them. But also, this is a simple module, so like, don't do that.

Reply:
a) yeah don't do that, but; b) might as well, won't hurt. I have amended this commit (and also fixed a thing in the docs rik noticed).